week-1/exercises/random.js

```diff
@@ -4,6 +4,10 @@ const maximum = 100;
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
 // In this exercise, you will need to work out what num represents?
+      
 // Try breaking down the expression and using documentation to explain what it means
+    // the num variable first find random number 1 up to 100 then round the number
+    // and multiplay by maximum -minimum and add one finally add minimum
```

Comment:
That is right. Also, think about the reason and logic behind doing them and how it works to get what we intended.